.env.example

```diff
@@ -14,17 +14,20 @@ REDIS_HOST=
 REDIS_PORT=
 
 # SMTP
+ENABLE_MAILGUN=
 SMTP_HOST=
 SMTP_PORT=
 SMTP_USERNAME=
 SMTP_PASSWORD=
 
 # Mailgun
+ENABLE_MAILGUN=
```

Comment:
By same I mean here also it is ENABLE_MAILGUN

Reply:
Fixed name